packages/react-core/src/next/components/Select/SelectOption.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import { MenuItemProps, MenuItem } from '../../../components/Menu';
+
+export interface SelectOptionProps extends Omit<MenuItemProps, 'ref'> {
+  /** Anything which can be rendered in a select option */
+  children?: React.ReactNode;
+  /** Classes applied to root element of select option */
+  className?: string;
+  /** Identifies the component in the Menu onSelect or onActionClick callback */
+  itemId?: any;
+  /** Flag indicating the option has a checkbox */
+  hasCheck?: boolean;
+  /** Render option as disabled */
+  isDisabled?: boolean;
+  /** Flag indicating if the option is selected */
+  isSelected?: boolean;
+  /** Flag indicating the option is focused */
+  isFocused?: boolean;
+}
+
+export const SelectOption: React.FunctionComponent<MenuItemProps> = ({
+  children,
+  className,
+  ...props
```

Comment:
Should the rest of the props be included here instead of just the spread props?

Reply:
They could but they would just be remapped to matching prop names on the Menu component. Spreading is just shorthanding `itemId={itemId} hasCheck={hasCheck} isDisabled={isDisabled}`, etc. LMK if we want to list the out instead.

On a side note about props, I don't list all of the Menu props, only the ones I used to make the current examples. But technically all Menu props will work because these extend the Menu props. I'm not sure if there's a clean way to indicate that though beyond looking at the Select prop interface.

Reply:
I don't think I have a strong opinion either way. I was more curious since the other components had listed other props vs spreading.